test/Interop/Cxx/operators/Inputs/member-out-of-line.h

```diff
@@ -0,0 +1,9 @@
+#ifndef TEST_INTEROP_CXX_OPERATORS_INPUTS_MEMBER_OUT_OF_LINE_H
+#define TEST_INTEROP_CXX_OPERATORS_INPUTS_MEMBER_OUT_OF_LINE_H
+
+struct IntBox {
+  int value;
+  IntBox operator+(IntBox rhs);
```

Comment:
Could you add a comment that this method being non-const is intentional?

Reply:
It's not intentional. It just doesn't matter. I can make it `const` if you prefer.